docs/run_test/troubleshoot_failures.rst

```diff
@@ -0,0 +1,184 @@
+Troubleshoot Test Failures
+=======================
+
+-  `Overview <#overview>`__
+-  `Test results <#test-results>`__
+-  `Console output <#console-output>`__
+-  `Log Folder Structure <#log-folder-structure>`__
```

Comment:
Log folder structure should be a sub-topic of Log files.

Reply:
I added Log Files as the header and made log folder structure a sub-header